src/wiki/forms.py

```diff
@@ -21,6 +18,10 @@
 from wiki.core.plugins.base import PluginSettingsFormMixin
 from wiki.editors import getEditor
 
+from .forms_account_handling import (  # noqa: ignore=F401 importing additional forms here as a convenience for other modules expecting them here
```

Comment:
I'm not sure I understand the necessity of this linting bypass?

Reply:
[wiki/views/accounts.py](https://github.com/django-wiki/django-wiki/blob/22a432d0940189de5852ea1b07bdfe1dd83950b3/src/wiki/views/accounts.py) refers to wiki.forms.UserCreationForm

Having split some forms into forms_account_handling to make their processing more readable, I thought it would still be nice to give the appearance that forms were still in wiki.forms. The lintier complains because those imports are not used in the module.

We could alternatively update references elsewhere in the project to point to wiki.forms_account_handling instead of wiki.forms.

Reply:
what would be your preference @benjaoming ?

Reply:
Let's declare an `__all__` in `forms.py`. This way, we won't have unintended breakage from re-organizing imports.

https://stackoverflow.com/questions/44834/can-someone-explain-all-in-python